src/block-management/block-toolbar.js

```diff
@@ -58,18 +58,19 @@ export class BlockToolbar extends Component<PropsType> {
 				>
 					<Toolbar>
 						<ToolbarButton
-							label={ __( 'Add block' ) }
+							title={ __( 'Add block' ) }
 							icon={ ( <Dashicon icon="plus-alt" style={ styles.addBlockButton } color={ styles.addBlockButton.color } /> ) }
 							onClick={ onInsertClick }
+							extraProps={ { hint: __( 'Double tap to add a block' ) } }
```

Comment:
As we talked with @JavonDavis , we will try to change this on a future PR and pass directly an accessibilityHint prop.
For now I prefer to keep it simpler.

Reply:
May I ask why the hint is "double tap"? A single tap seems to be enough to execute the action here, is it not?

Reply:
When VoiceOver is active, a single tap will select an element, and a double tap will trigger the action. It's kind of standard for hints.

Reply:
Oh I see, indeed I needed to actually enable TalkBack to see it in action 👍